src/utilities/activity-utils.ts

```
@@ -214,10 +214,17 @@ export const removeDuplicateUrls = (urls: string[]) => urls.filter((url, index)
 
 export const getAllUrlsInActivity = async (activity: Activity, urls: string[] = []) => {
   const addExternalUrls = (object: any) => {
-    walkObject(object, (s) => {
+    walkObject(object, (s, key) => {
       if (isExternalOrModelsResourcesUrl(s)) {
         urls.push(s);
       }
+      // add all external urls in the stringified author data json
+      if (key === "author_data") {
+        try {
+          const authorData = JSON.parse(s);
+          addExternalUrls(authorData);
+        } catch (e) {} // eslint-disable-line no-empty
```

You can also just put a comment like `// ignore errors` inside the braces.